script/update

```diff
@@ -252,38 +248,16 @@ def setup_mips64el_toolchain(target_arch):
   subprocess.check_call(['chromium/scripts/copy_config.sh'], cwd=ffmpeg_dir)
   subprocess.check_call(['chromium/scripts/generate_gn.py'], cwd=ffmpeg_dir)
 
-def run_gn(target_arch):
-  env = os.environ.copy()
-  if sys.platform in ['win32', 'cygwin']:
-    env['DEPOT_TOOLS_WIN_TOOLCHAIN'] = '0'
-
-  target_os = {
-    "cygwin": "win",
-    "darwin": "mac",
-    "linux2": "linux",
-    "win32": "win",
-  }.get(sys.platform)
 
-  if target_arch == 'ia32':
-    target_cpu = 'x86'
-  else:
-    target_cpu = target_arch
+def run_gn(target_arch):
+  gn_args = {}
 
-  for component in COMPONENTS:
-    # Create output dir.
-    output_dir = get_output_dir(SOURCE_ROOT, target_arch, component)
-    if not os.path.isdir(output_dir):
-      os.makedirs(output_dir)
-
-    # Create a GN build configuration file.
-    args_file_path = os.path.join(CHROMIUMCONTENT_SOURCE_DIR, 'args', component + '.gn')
-    with open(args_file_path) as f:
-      gn.create_args(output_dir, f.read(),
-          target_os=target_os,
-          target_cpu=target_cpu)
-
-    # Generate Ninja build files.
-    gn.generate(output_dir, chromium_root_dir=SRC_DIR, depot_tools_dir=DEPOT_TOOLS, env=env)
+  script_path = os.path.join(SOURCE_ROOT, 'script', 'run-gn')
+  script_arguments = [
+      '--args', json.dumps(gn_args),
+      '--target_arch', target_arch
+      ]
+  return subprocess.call([sys.executable, script_path] + script_arguments)
```

Comment:
why a subprocess rather than a function call?

Reply:
Who knows ¯\_(ツ)_/¯

Reply:
It would be really easy! Just rename `run-gn` to `run_gn.py`, then `from run_gn import main as run_gn` and change `main` to accept argv as a parameter. Even better would be to factor out the functionality you want to access as a function with sensible arguments and call it, e.g. `def generate_all_build_files(components=COMPONENTS)`.